src/components/content/order/migrate/DeploymentForm.tsx

```diff
@@ -148,3 +149,15 @@ export const DeploymentForm = ({
         </div>
     );
 };
+
+DeploymentForm.propTypes = {
```

Comment:
I think this warning is coming because the variable name in line 39 is called `props`. Can you please rename it to something else.. Eg., `deployParams`